platform/ios/PolyPodApp/PodApi/Endpoint/Endpoint.swift

```diff
@@ -12,17 +13,19 @@ protocol EndpointDelegate {
 
 struct EndpointInfo: Decodable {
     let url: String
-    let auth: String
+    let auth: String?
 }
 
 final class Endpoint: EndpointProtocol {
     
-    init() {
+    init(allowInsecure: Bool = false) {
+        self.allowInsecure = allowInsecure
         delegate = nil
     }
     
+    var allowInsecure: Bool
     var delegate: EndpointDelegate?
-    let network: Network = Network()
+    let network: Network = Network(allowInsecure)
```

Comment:
Maybe just inject Network, so need to pass intermediary data to the Endpoint. Consider this:
```swift
init(network: NetworkProtocol) {
         self.network = network
         delegate = nil
     }
let network: NetworkProtocol
````

Reply:
To be able to have a testing feature we figured that we would need to put allowInsecure as part of endpoints.json and then also in the decodable and go from there. The other would only be valid in local testing in ios